sections/Abstract.tex

```diff
@@ -1 +1,8 @@
-Abstract
\ No newline at end of file
+We investigate whether using a topic model can improve \gls{ir} methods.
+We employ a probabilistic topic model, specifically \gls{lda}, to automatically extract topics from Danish news articles.
+%How can \gls{pr} be used on a dataset without explicit edges?
+We use the topic distribution, from the \gls{lda}, to construct the adjacency matrix edge by measuring the Jensen-Shannon distance between each article topic distribution, since references between news articles do not exist.
+We use \gls{lda}, \gls{tf-idf}, \gls{bm25}, \gls{lm} and \gls{pr} as our baselines for testing \gls{ir} performance.
+We also combine these in various ways to investigate the individual performance of each method.
+We evaluate the \gls{ir} methods by generating document and topic queries which are based on the topic distribution from the \gls{lda}.
+\gls{bm25} is the best performing baseline, we have, and we improve \gls{bm25} by combining it with \gls{pr}.
```

Comment:
Jeg tror gerne at jeg først vil snakke om hvordan vi prøver på at evaluere på to måder / en ny måde og først derefter slutte af med algoritmerne.
Men har svært ved at få det formuleret...

Reply:
```suggestion
We test various \gls{ir} methods' ability to find documents related to a query, but also to find documents related to topics expressed in queries.
We then evaluate \gls{ir} performance of combinations of \gls{lda}, \gls{tf-idf}, \gls{bm25}, \gls{lm} and \gls{pr}.
```